### PR TITLE
docs(templating): migrate template engines to the community hub

### PR DESCRIPTION
**Pull request by tasso94**
The following template engines wrapper were moved to the community hub:

* Saxon XSLT
* Saxon XQuery
* Apache Velocity

related to camunda/camunda-bpm-platform#3695, camunda/camunda-bpm-platform#3696, camunda/camunda-bpm-platform#3697

### CODE REVIEW

**Review by yanavasileva — COMMENTED** [review]
👍 Overall looks good, please check two comments below.

File: content/update/minor/719-to-720/_index.md [hunk 1]
```diff
@@ -224,3 +225,38 @@ The Camunda Docker images are based on Alpine. This release updates the Alpine b
...
+</dependency>
+
+<dependency>
+  <groupId>org.camunda.community.template.engine</groupId>
+  <artifactId>camunda-7-template-engine-xslt</artifactId>
+  <version>2.2.0</version>
+</dependency>
+```
+
+We are looking for maintainers for the template engine extensions. Feel free to reach out to us [via the forum](https://forum.camunda.io/c/camunda-platform-7-topics/39) if you are interested.
```
[the remark this answers is not in the document]
Comment (tasso94) [reply content/update/minor/719-to-720/_index.md:262]: I wonder if it is easy to find maintainers. The migration guide is read by a considerable fraction of the user base, and there might be people who use the template engines in production who won't find the forum post. I prefer to keep this paragraph and remove it once maintainers are found.

File: content/user-guide/process-engine/templating.md
[the remark this answers is not in the document]
Comment (tasso94) [reply content/user-guide/process-engine/templating.md:1]: I raised a PR: https://github.com/camunda/camunda-docs-static/

[the remark this answers is not in the document]
Comment (tasso94) [reply content/user-guide/process-engine/templating.md:1]: Done. Thank you 🙂 👍

**Review by yanavasileva — APPROVED** [review]
👍

**Pull request merged**